src/Avalonia.Input/Gestures.cs

```diff
@@ -6,6 +6,7 @@ namespace Avalonia.Input
 {
     public static class Gestures
     {
+        private static bool _isDoubleTapped = false;
```

Comment:
That's a _global_ variable. We usually try to avoid those

Reply:
Device and input state information must usually be maintained and managed centrally. It might help to suggest an alternative. Otherwise, I expect to see things like this myself.

Reply:
The app might run on 2 independent touch screens. Or even on 1 touch screen with 2 windows on it. Because of such global variables we'll misinterpret tap sequences caused by the user tapping on those windows sequentially.

Reply:
We decided to temporarily leave it as is

Reply:
> Nothing is more permanent than temporary


Yes, the original static variable in that file dates back to 2016 when DoubleTapped event was introduced. So we aren't introducing any new problems.